src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/ModuleBacktrack.kt

```diff
@@ -306,13 +312,35 @@ object ModuleBacktrack : Module("Backtrack", Category.COMBAT) {
     fun isLagging() =
         enabled && packetQueue.isNotEmpty()
 
-    private fun shouldBacktrack(target: Entity) =
-        target.shouldBeAttacked() &&
-            target.boxedDistanceTo(player) in range &&
+    init {
+        handler<AttackEvent> {
+            val enemy = it.enemy
+            shouldPause = enemy is LivingEntity && enemy.hurtTime < 10
+        }
```

Comment:
What on earth? Why is it here? Sure, that works too, but we agreed to use variables to make it act like a function.

Reply:
when i moved it to the existing attack handler it was mostly just returning false, it works on here for some reason

Reply:
yeah because there is already an attack handler??